sass/components/_mixins.scss

```diff
@@ -2,4 +2,17 @@
 //     -webkit-box-shadow: $args1, $args2;
 //     -moz-box-shadow: $args1, $args2;
 //     box-shadow: $args1, $args2;
-// }
\ No newline at end of file
+// }
+
+@mixin not-selectable {
+	-webkit-touch-callout: none;
```

Comment:
Those browser specific prefixes were removed. Also I have renamed mixins to be more inline with your naming conventions.